src/osm_configurator/model/project/calculation/paralellization/work_manager.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import annotations
+
+import multiprocessing
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from typing import Any
+    from typing import List
+    from typing import Tuple
+    from typing import Callable
+    from src.osm_configurator.model.project.calculation.paralellization.work import Work
+    from multiprocessing import Pool
+
+
+class WorkManager:
+    """
+    The WorkManager allow a parallelization of work via multiprocessing.
+    """
+    def __init__(self, max_processes: int):
+        """
+        Creates an empty work manager.
+        Args:
+            max_processes (int): The maximal number of processes that might be run at the same time
+        """
+        self._max_workers = max_processes
+        self._work_to_do: List[Work] = []
+
+    def append_work(self, work: Work):
+        """
+        Adds a piece of work to the manager
+        """
+        self._work_to_do.append(work)
+
+    def do_all_work(self) -> List:
```

Comment:
List maybe be more specifc in typehint which list

Reply:
the list can not be more specific, because the result of the task can be everything